Plugins/BenchmarkCommandPlugin/BenchmarkCommandPlugin.swift

```diff
@@ -281,6 +285,19 @@ import Glibc
             }
         }
 
+        if !skipLoadingBenchmarksFlagIsValid {
+            print("")
+            print(
+                "Flag --skip-loading-benchmark-targets is only valid for 'thresholds check' operations."
```

Comment:
As discussed in #327 and in other older issues, the only reason you might need to build benchmark targets in `thresholds check` command is to load the threshold-tolerances that are in code in `configuration.thresholds`.
For users that have moved to using static threshold files with `range`/`relative`, this is pointless and only a waste of time.
So I've added the option to skip building benchmark targets for `thresholds check`.

Reply:
Checking the benchmark runs, this has reduced the "compare" step's runtime in CI from 30 to 15s, so a 15s saving.
The package itself is very lightweight. Only that it has 2 benchmark targets to build, which isn't abnormal but does mean 2 targets need to get built.

Reply:
In a pretty chunky project that step takes 68s for 1 benchmark target. I'd assume at least 30s of that will be shaved off with this change.

That CI takes 18 minutes though. These are all nice savings but aren't a big deal.